spec/paperclip/deprecations_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+require 'aws-sdk'
+require 'active_support/testing/deprecation'
+
+describe Paperclip::Deprecations do
+  include ActiveSupport::Testing::Deprecation
+
+  describe '.check' do
+    before do
+      ActiveSupport::Deprecation.silenced = false
+    end
+
+    context 'when active record version is < 4.2' do
+      it 'displays deprecation warning' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.